src/OpenTK/Platform/X11/X11DisplayDevice.cs

```diff
@@ -144,35 +161,44 @@ private bool QueryXinerama(List<DisplayDevice> devices)
                         dev.IsPrimary = true;
                         first = false;
                     }
-                    devices.Add(dev);
                     // It seems that all X screens are equal to 0 is Xinerama is enabled, at least on Nvidia (verify?)
-                    deviceToScreen.Add(dev, 0 /*screen.ScreenNumber*/);
+                    dev.ScreenIndex = screen.ScreenNumber;
+
+                    devices.Add(dev);
                 }
             }
```

Comment:
Leave a space after the final closing brace. It sets the return apart from the rest of the code.